### DISCUSSION TITLE
Getting the list of selected files in a relative format

### DISCUSSION THREAD

**Discussion by Archie-2021** [gokcehan/lf#1369] [Q&A] [answered]
I am trying to create a function in lf to feed the selected file to `qmv`, [link](https://www.nongnu.org/renameutils/). It is a cool utility that lets you rename file in a vim buffer with filenames in each line. The problem is that it uses `ls` to the input and ls only provides paths in relative format, so `qmv $fx` would not work properly. Is there a variable or some trick to solve this issue?

Thanks

**Comment by joelim-work** [comment, marked as answer]
`$fx` contains absolute paths. If you want relative paths you can use `realpath`, by replacing `$fx` with `$(realpath --relative-to=. $fx)`.

Interestingly `qmv $fx` works for me (the files are displayed as absolute paths though), so I'm not sure what your issue actually is.

**Reply by Archie-2021** [reply]
Thank you, I kinda forget about this thread and find the `realpath` as the solution by myself too. 

`qmv $fx` is going to open up the editor by it is not going to take effect after saving it. I think `qmv` and `lf` are very cool to be used together. I am just starting to find cool programs like this to use with `lf`. Would like to see more show cases of other people's workflow.

**Reply by joelim-work** [reply]
No worries.

If you haven't seen it before, you can have a look at the [wiki](https://github.com/gokcehan/lf/wiki) which contains many tips from other users. Feel free to add your own examples there too.
